Utils.cs

```diff
@@ -6,15 +6,13 @@
 
 namespace GettingStartedWithCSharp
 {
-    public class Utils
+    class Utils : IUtils
```

Comment:
Daca ti-ai facut treaba cum trebuie, tot ce e aici trebuie sa devina niste metode private in Presenter, caci momentan nu ar trebui sa fie folosite in alta parte.